packages/date-picker/src/CalendarNav.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import styles from "./DatePicker.scss"
+
+export type CalendarNavProps = {
+  onPreviousClick: () => void
+  onNextClick: () => void
+}
+
+export const CalendarNav: React.FunctionComponent<CalendarNavProps> = ({
+  onPreviousClick,
+  onNextClick,
+}) => (
+  <div className={styles.navbar}>
+    <button
+      className={styles.arrows}
```

Comment:
There's nothing on these elements to indicate to a screen reader what these buttons do. Good use case for `aria-label`

Reply:
Good catch, will update